src/content-helper/common/utils/functions.ts

```diff
@@ -50,3 +50,62 @@ export async function isEditorReady(): Promise<void> {
 		} );
 	} );
 }
+
+/**
+ * Adds ITM parameters to a URL.
+ *
+ * @since 3.18.0
+ *
+ * @param {string} url                The URL to add ITM parameters to.
+ * @param {Object} itmParams          The ITM parameters to add to the URL.
+ * @param {string} itmParams.campaign The campaign parameter.
+ * @param {string} itmParams.source   The source parameter.
+ * @param {string} itmParams.medium   The medium parameter.
+ * @param {string} itmParams.content  The content parameter.
+ * @param {string} itmParams.term     The term parameter.
+ *
+ * @return {string} The URL with ITM parameters added.
+ */
+export function addITMParamsToURL( url: string, itmParams: {
+	campaign: string;
+	source?: string;
+	medium?: string;
+	content?: string;
+	term?: string;
+} ): string {
+	const urlObj = new URL( url );
+	urlObj.searchParams.set( 'itm_campaign', itmParams.campaign );
+
+	if ( itmParams.source ) {
+		urlObj.searchParams.set( 'itm_source', itmParams.source );
+	}
+	if ( itmParams.medium ) {
+		urlObj.searchParams.set( 'itm_medium', itmParams.medium );
+	}
+	if ( itmParams.content ) {
+		urlObj.searchParams.set( 'itm_content', itmParams.content );
+	}
+	if ( itmParams.term ) {
+		urlObj.searchParams.set( 'itm_term', itmParams.term );
+	}
+	return urlObj.toString();
+}
+
+/**
+ * Removes ITM parameters from a URL.
+ *
+ * @since 3.18.0
+ *
+ * @param {string} url The URL to remove ITM parameters from.
+ *
+ * @return {string} The URL with ITM parameters removed.
+ */
+export function removeITMParamsFromURL( url: string ): string {
+	const urlObj = new URL( url );
+	urlObj.searchParams.delete( 'itm_campaign' );
+	urlObj.searchParams.delete( 'itm_source' );
+	urlObj.searchParams.delete( 'itm_medium' );
+	urlObj.searchParams.delete( 'itm_content' );
+	urlObj.searchParams.delete( 'itm_term' );
+	return urlObj.toString();
```

Comment:
Let's add an empty line here.

```suggestion

	return urlObj.toString();
```